appinventor/components/src/com/google/appinventor/components/runtime/Image.java

```diff
@@ -173,4 +173,28 @@ public void ScalePictureToFit(boolean scale) {
   public void Animation(String animation) {
     AnimationUtil.ApplyAnimation(view, animation);
   }
+
+  @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_SCALING,
+      defaultValue = Component.SCALING_SCALE_PROPORTIONALLY + "")
+  @SimpleProperty(description = "This allows you to scale the picture in " +
```

Comment:
We should internationalize new tooltips since we'll be going back later an internationalizing them all.   See the note on internationalization in How to Add a Component.

Also change the text to make it more clear (add there are now only two choices, not three).

This property determines how the picture scales according to the Height or Width of the Image.   Scale proportionally (0) preserves the picture aspect ratio.  Scale to fit (1) matches the Image area, even if the 
aspect ratio changes.

Don't forget to add this to the web documentation as described in How to Regenerate the App Inventor Documentation.
Also feel free to add the Simplified or Traditional Chinse localizations for the new strings your' added.